MessagesView/MessagesGroupingCoordinator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MessagesGroupingCoordinator.swift
+//  MessagesView
+//
+//  Created by Damian Kanak on 11/04/17.
+//  Copyright © 2017 pgs-dkanak. All rights reserved.
+//
+
+import Foundation
+
+class MessagesGroupingCoordinator {
+    public func selectBackgroundFor(index: Int, inMessages messages: [MessagesViewChatMessage], withBubble bubbbleImage: BubbleImage)->UIImage {
```

Comment:
This function can be moved to MessageView, no need for separate class with no properties.